### ISSUE TITLE
NeurophException is type RuntimeException

### ISSUE THREAD

**Issue by leviem1** [neuroph/neuroph#62]
Because the Javadoc does not list `NeuralNetwork.createFromFile()` as throwing any exceptions and the exceptions that are thrown are `RuntimeExceptions`, it becomes difficult to know when to catch exceptions. For example, it may be assumed that running `NeuralNetwork.createFromFile()` on a nonexistent file will not produce an exception since the compiler does not ask the developer to catch any exceptions. This makes debugging difficult when trying to catch all instances of `NeurophException` since they are never documented or caught by the compiler.

**Comment by neuroph** [comment]
Yes, you're right. I'll make the changes you proposed since it will require fixing in many places.
Cheers
